### PR TITLE
Changed string id/variable name to reflect usage.

### PR DESCRIPTION
**Pull request by opakedragon**
Changed string id/variable name to reflect usage. The Player, not the Pokémon, picked up money (¥), not an item.

## Description
rename:
STRINGID_PKMNPICKEDUPITEM to STRINGID_PLAYERPICKEDUPMONEY
sText_PkmnPickedUpItem to sText_PlayerPickedUpMoney

## **Discord contact info**
Opake DraGon#8094

### CODE REVIEW

File: include/constants/battle_string_ids.h [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #ifndef GUARD_CONSTANTS_BATTLE_STRING_IDS_H
 #define GUARD_CONSTANTS_BATTLE_STRING_IDS_H
 
-#define BATTLESTRINGS_COUNT     369
+#define BATTLESTRINGS_COUNT     371
```
Comment (GriffinRichards) [inline include/constants/battle_string_ids.h:4]: Revert

File: include/constants/battle_string_ids.h [hunk 2]
```diff
@@ -382,5 +382,7 @@
 #define STRINGID_PKMNBOXLANETTESPCFULL      378
 #define STRINGID_TRAINER1WINTEXT            379
 #define STRINGID_TRAINER2WINTEXT            380
+#define STRINGID_PKMNPICKEDUPITEM           381
+#define STRINGID_PKMNPICKEDUPITEMS          382
```
Comment (GriffinRichards) [inline include/constants/battle_string_ids.h:386]: Unnecessary

File: src/battle_message.c [hunk 3]
```diff
@@ -180,7 +180,7 @@ static const u8 sText_PkmnStayedAwakeUsing[] = _("{B_DEF_NAME_WITH_PREFIX} staye
 static const u8 sText_PkmnStoringEnergy[] = _("{B_ATK_NAME_WITH_PREFIX} is storing\nenergy!");
 static const u8 sText_PkmnUnleashedEnergy[] = _("{B_ATK_NAME_WITH_PREFIX} unleashed\nenergy!");
 static const u8 sText_PkmnFatigueConfusion[] = _("{B_ATK_NAME_WITH_PREFIX} became\nconfused due to fatigue!");
-static const u8 sText_PkmnPickedUpItem[] = _("{B_PLAYER_NAME} picked up\n¥{B_BUFF1}!\p");
+static const u8 sText_PlayerPickedUpMoney[] = _("{B_PLAYER_NAME} picked up\n¥{B_BUFF1}!\p"); // The Player, not the Pokémon, picked up money (¥), not an item.
```
Comment (GriffinRichards) [inline src/battle_message.c:183]: Remove the comment here and in `gBattleStringsTable`. They're only clarifying from the previous name, not needed now

**Review by GriffinRichards — CHANGES_REQUESTED** [review]

File: include/constants/battle_string_ids.h (hunk 2 above)
Comment (opakedragon) [reply include/constants/battle_string_ids.h:386]: this was an accident i am trying to undo. I meant for this to be in a new branch

**Pull request closed without merging**